### ISSUE TITLE
Add localization

### ISSUE THREAD

**Issue by HoLuLuLu** [hughesjs/FitnessMachine#100]
At the moment the whole app is in English. This is fine for the beginning but it would be cool to have the texts in other languages to offer this app to non native/English speaking people.

**Comment by hughesjs** [comment]
Absolutely, but to be honest, I just want to get the features in first then can worry about that
